content/en/docs/instrumentation/js/libraries.md

```diff
@@ -14,253 +51,221 @@ For example,
+update your `instrumentation.ts` (or `instrumentation.js`) as follows:
 
-<!-- textlint-disable -->
-<!-- markdownlint-disable -->
 <!-- prettier-ignore-start -->
 {{< tabpane langEqualsHeader=true >}}
 
 {{< tab TypeScript >}}
-/* tracing.ts */
-
-// Import dependencies
-import { getNodeAutoInstrumentations } from "@opentelemetry/auto-instrumentations-node";
-import opentelemetry from "@opentelemetry/api";
-import { Resource } from "@opentelemetry/resources";
-import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";
-import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";
-import { registerInstrumentations } from "@opentelemetry/instrumentation";
-import { ConsoleSpanExporter, BatchSpanProcessor } from "@opentelemetry/sdk-trace-base";
-
-// This registers all instrumentation packages
-registerInstrumentations({
-  instrumentations: [
-    getNodeAutoInstrumentations()
-  ],
-});
+/*instrumentation.ts*/
+...
+import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
 
-const resource =
-  Resource.default().merge(
-    new Resource({
-      [SemanticResourceAttributes.SERVICE_NAME]: "service-name-here",
-      [SemanticResourceAttributes.SERVICE_VERSION]: "0.1.0",
-    })
-  );
-
-const provider = new NodeTracerProvider({
-    resource: resource,
+const sdk = new NodeSDK({
+  ...
+  instrumentations: [getNodeAutoInstrumentations()]
 });
-const exporter = new ConsoleSpanExporter();
-const processor = new BatchSpanProcessor(exporter);
-provider.addSpanProcessor(processor);
 
-provider.register();
+sdk.start()
+
 {{< /tab >}}
 
 {{< tab JavaScript >}}
-/* tracing.js */
+/*instrumentation.js*/
+const { getNodeAutoInstrumentations } = require('@opentelemetry/auto-instrumentations-node');
 
-// Require dependencies
-const { getNodeAutoInstrumentations } = require("@opentelemetry/auto-instrumentations-node");
-const opentelemetry = require("@opentelemetry/api");
-const { Resource } = require("@opentelemetry/resources");
-const { SemanticResourceAttributes } = require("@opentelemetry/semantic-conventions");
-const { NodeTracerProvider } = require("@opentelemetry/sdk-trace-node");
-const { registerInstrumentations } = require("@opentelemetry/instrumentation");
-const { ConsoleSpanExporter, BatchSpanProcessor } = require("@opentelemetry/sdk-trace-base");
-
-// This registers all instrumentation packages
-registerInstrumentations({
+const sdk = new NodeSDK({
+  ...
+  // This registers all instrumentation packages
   instrumentations: [
-    getNodeAutoInstrumentations()
-  ],
-});
-
-const resource =
-  Resource.default().merge(
-    new Resource({
-      [SemanticResourceAttributes.SERVICE_NAME]: "service-name-here",
-      [SemanticResourceAttributes.SERVICE_VERSION]: "0.1.0",
-    })
-  );
-
-const provider = new NodeTracerProvider({
-    resource: resource,
+    // Express instrumentation expects HTTP layer to be instrumented
```

Comment:
Looks like these two sections got switched on the JavaScript tab - `getNodeAutoInstrumentations()` should be here, and http/express should be down on line 168.

Reply:
fixed :-)